package.json

```diff
@@ -23,18 +24,20 @@
     "node": ">= 8.x"
   },
   "scripts": {
-    "test": "npm run prettier:check && npm run lint && npm run check && npm run testonly && npm run check:ts",
-    "test:ci": "yarn check --integrity && npm run prettier:check && npm run lint -- --no-cache && npm run check && npm run testonly:cover && npm run check:ts && npm run build",
-    "testonly": "mocha --full-trace src/**/__tests__/**/*-test.js",
+    "test": "npm run prettier:check && npm run lint && npm run check && npm run testonly",
+    "test:ci": "yarn check --integrity && npm run prettier:check && npm run lint -- --no-cache && npm run check && npm run testonly:cover && npm run build",
+    "testonly": "mocha -r ts-node/register --full-trace src/**/__tests__/**/*-test.js src/**/__tests__/**/*-test.ts ",
```

Comment:
Can you please move `ts-node/register` into
https://github.com/graphql/graphql-js/blob/master/.mocharc.yml
I'm not sure but maybe `mocha` we can the same format as `prettier` command so you can write `"src/**/__tests__/**/*-test.{js,ts}"`.
Can you please test it?